### PR TITLE
Use apache instead of PHP builtin server

### PR DESCRIPTION
**Pull request by alexpott**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -34,13 +65,43 @@ done
...
-    cp --dereference --recursive "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
+# Copy Thunder project to Dockerfile context if project path is provided
+if [ "${PROJECT_PATH}" != "" ]; then
+  if [ "${OS_NAME}" == "osx" ]; then
+      rm -rf "${SCRIPT_DIRECTORY}/www"
+      cp -R "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
+  else
+      rm --recursive --force  "${SCRIPT_DIRECTORY}/www"
+      cp --dereference --recursive "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
+  fi
```
Comment (dbosen) [inline build.sh:76]: As discussed, let's simplify all if/else's by just using shorthand syntax, which runs on every OS

File: Dockerfile [hunk 2]
```diff
@@ -44,35 +36,51 @@ RUN set -xe; \
...
+# Install Elastic APM - this is run as a shell script so we can install in the
+# doc root.
+RUN set -xe; \
     \
-    echo -e "\nexport PATH=\"\$PATH:/home/thunder/.composer/vendor/bin\"\n" >> /home/thunder/.profile; \
+    su - thunder --command="install-elastic-apm";
+
+# Provision core tests
+# @todo move this to one of our modules only work with core checkout atm.
+RUN set -xe; \
```
Comment (dbosen) [inline Dockerfile:79]: Each RUN statement increases the docker image size, we could combine the last three RUN  statements.

File: test/travis-ci-run.sh [hunk 3]
```diff
@@ -22,26 +22,33 @@ until curl --output /dev/null --silent --head --fail "http://localhost:8080/"; d
...
+if [ "${PROFILE}" == "thunder" ]; then
+  # Check that relevant modules are installed
+  docker exec "${TEST_THUNDER_PHP_DOCKER_ID}" su - thunder --command='cd "${DOC_ROOT}"; drush pm:list --status=enabled --format=json;' | jq --exit-status '.thunder_performance_measurement, .media_entity_generic'
 
-# Workaround for testsite_builder module name until Drush is fixed
-# TODO: Move check back to "drush pm:list" command when Drush issue is fixed (https://github.com/drush-ops/drush/issues/4182)
-docker exec "${TEST_THUNDER_PHP_DOCKER_ID}" su - thunder --command='cd /home/thunder/www/docroot; /home/thunder/www/vendor/drupal/console/bin/drupal debug:module --status=enabled;' | grep --silent "testsite_builder"
+  # Workaround for testsite_builder module name until Drush is fixed
+  # TODO: Move check back to "drush pm:list" command when Drush issue is fixed (https://github.com/drush-ops/drush/issues/4182)
+  docker exec "${TEST_THUNDER_PHP_DOCKER_ID}" su - thunder --command='cd "${DOC_ROOT}"; /home/thunder/www/vendor/drupal/console/bin/drupal debug:module --status=enabled;' | grep --silent "testsite_builder"
```
Comment (chrfritsch) [inline test/travis-ci-run.sh:36]: I am DOC ROOT 👨🏻‍⚕️

File: build.sh (hunk 1 above)
Comment (dbosen) [inline build.sh:76]: As discussed, let's simplify all if/else's by just using shorthand syntax, which runs on every OS

Comment (alexpott) [reply build.sh]: I've done this in the latest commit. The one thing we can't have as far as I can see is the --deference option. Doesn't exist on OSX.

Comment (alexpott) [reply build.sh]: Nothing broke so yay for less complexity.

**Pull request merged**